src/deploy/functions/build.ts

```diff
@@ -309,7 +320,7 @@ export function resolveBackend(build: Build, userEnvs: Record<string, string>):
         "environmentVariables",
         "labels"
       );
-      // proto.copyIfPresent(bkEndpoint, endpoint, "secretEnvironmentVariables");
+      proto.copyIfPresent(bkEndpoint, endpoint, "secretEnvironmentVariables");
```

Comment:
nit (feel free to disregard) - what are your thoughts on naming these two endpoints `buildEp` and `backendEp`? I was a bit confused to what this generic `endpoint` was at first.

Reply:
Honestly, I think it's a great idea.